### ISSUE TITLE
broccoli is spelled wrong

### ISSUE THREAD

**Issue by CalamariAce** [mrmaxguns/wonderwordsmodule#22]
Your database's spelling of broccoli is wrong. It should have two c's instead of two l's.

**Comment by mrmaxguns** [comment]
You're right. The database has both spellings, but when I looked online, I couldn't find any use for the word `brocolli`, so I'll remove it.
